src/test/resources/logback-test.xml

```diff
@@ -14,7 +14,7 @@
         <appender-ref ref="CONSOLE" />
     </appender>
 
-    <root level="OFF">
+    <root level="ERROR">
```

Comment:
Why this change?

Reply:
I usually change logback test from OFF to ERROR in case of some test fail.
i forgot to change it back